### ISSUE TITLE
Show Installation and Run of the Ersilia Model Hub - Outreachy

### ISSUE THREAD

**Issue by GemmaTuron** [ersilia-os/ersilia#36]
This issue is for Outreachy Applicants interested in contributing to the Project "Improve the Ersilia Model Hub, a FOSS platform offering pre trained AI/ML models for research"

**Before** tackling any coding issue, comment here with:
 - Your name
 - System (OS, Windows., Linux) and version
 - Your background in Python (required skill for this project)
 - A snapshot of your CLI showing that you have successfully run a model

This is required before we accept any pull request for the issues labelled with the tag "code"

**Comment by Riyabelle25** [comment]
- Hi, I'm **Riya Elizabeth John** a third-year B.E undergrad at IIT Roorkee, India
- **System (OS, Windows., Linux) and version:** **MacOS** Monterey Version 12.1, Apple M1 chip
- **Your background in Python (required skill for this project):** I'm primarily a Python Developer with (both open-source and industrial) experience working in ML Toolboxes/Packages. As part of the Vision and Language Group here at my university, I also have a sound knowledge of Deep-learning models and Machine-learning algorithms. In short, I really love working on software platforms that intersect with the ML domain.
- **A snapshot of your CLI showing that you have successfully run a model:**
<img width="1440" alt="Screenshot 2022-03-28 at 1 09 30 PM" src="https://user-images.githubusercontent.com/55790848/160349735-d2eb35a2-aff2-4a5d-9f4a-8dcb54ae9c61.png">

**Comment by GemmaTuron** [comment]
> * Hi, I'm **Riya Elizabeth John** a third-year B.E undergrad at IIT Roorkee, India
> * **System (OS, Windows., Linux) and version:** **MacOS** Monterey Version 12.1, Apple M1 chip
> * **Your background in Python (required skill for this project):** I'm primarily a Python Developer with (both open-source and industrial) experience working in ML Toolboxes/Packages. As part of the Vision and Language Group here at my university, I also have a sound knowledge of Deep-learning models and Machine-learning algorithms. In short, I really love working on software platforms that intersect with the ML domain.
> * **A snapshot of your CLI showing that you have successfully run a model:**
> 
> <img alt="Screenshot 2022-03-28 at 1 09 30 PM" width="1440" src="https://user-images.githubusercontent.com/55790848/160349735-d2eb35a2-aff2-4a5d-9f4a-8dcb54ae9c61.png">

Hello @Riyabelle25, 
Many thanks for answering! Next step would be to pick one of the issues with the label "code" to start working on it.
@miquelduranfrigola will be your mentor for that, ping him in the issue of your interest and it will be assigned to you.
You can already record this issue in the Outreachy Website to start your application for this project.

_Please note that we are updating the GitHub workflows for contributors and this might take a few days to be up and running, so you will not be able to push the work just yet, work on your branch in the meantime_

**Comment by Riyabelle25** [comment]
Aight, will do 💛

**Comment by Prakashh21** [comment]
Hi, I'm Prakash, an undergrad from India
**System (OS, Windows., Linux) and version:** Ubuntu 20.04.4 LTS
**Your background in Python (required skill for this project):**  I have a fair bit of experience in python , I have worked with libraries including pandas and numPy , although I am new to opensource but I would love to be a part of this project and contribute. Thankyou !!
**A snapshot of your CLI showing that you have successfully run a model:**![Screenshot from 2022-03-28 14-34-20](https://user-images.githubusercontent.com/90280586/160364716-42cfe0b7-61ba-4b05-aaea-b622caf9c273.png)

**Comment by Rufaida98** [comment]
![Capture ercilia](https://user-images.githubusercontent.com/92082003/160389612-789e1781-3fbe-4125-8f05-0c24444bb0a4.PNG)
i installed succefully but i coudnt run the model. Can anyone help me ?

**Comment by Riyabelle25** [comment]
> ![Capture ercilia](https://user-images.githubusercontent.com/92082003/160389612-789e1781-3fbe-4125-8f05-0c24444bb0a4.PNG) i installed succefully but i coudnt run the model. Can anyone help me ?

Could you specify your OS setup, and reproduce your steps?

**Comment by Rufaida98** [comment]
![Os](https://user-images.githubusercontent.com/92082003/160394162-19bc900a-788c-4744-be1a-02d22230e289.PNG)
I did the steps again in anaconda power shell and gave me the same result

**Comment by Rufaida98** [comment]
hi @anamika-yadav99 yes i am trying installing wsl right now. wish me luck :)

**Comment by 1218muskan** [comment]
- Hi, my name is Muskan Gupta. I am a 2nd year B.Tech Undergraduate from India.
- **System (OS, Windows., Linux) and version:** Windows 10. Using WSL (Ubuntu 18.04 LTS).
- **Your background in Python (required skill for this project):** I am familiar with all the basics and some advanced concepts of python. I have learned and experimented with python libraries including numpy, pandas and some data visualization libraries including matplotlib and seaborn. I have recently started with my ML journey. Being a ML/AI enthusiast I would love to contribute in this project and enhance my knowledge.
-  **A snapshot of your CLI showing that you have successfully run a model:**
- 
![image](https://user-images.githubusercontent.com/73191789/160400942-97119e23-5651-4cc4-830a-e6afdd6f7a62.png)

**Comment by anamika-yadav99** [comment]
> @1218muskan i have windows 10 and i installed ubuntu but i faced a problem ![c ubuntu](https://user-images.githubusercontent.com/92082003/160408218-24f25c43-ed96-4e50-bf22-5be66320d398.PNG) Can you tell me hoe you run the program successfully?

you can follow this tutorial : https://www.minitool.com/news/enable-virtualization-windows-10.html

**Comment by anamika-yadav99** [comment]
Hi, I'm Anamika. I'm 3rd year undergraduate from India.
System (OS, Windows., Linux) and version:  Windows 10.  Using WSL (Ubuntu 18.04)
Your background in Python (required skill for this project): I'm a Python Developer with open source experience working in ML, bioinformatics, ML toolboxes/packages . I'm a student researcher working in application of ML algorithms in solving complex problems in biotechnology.
A snapshot of your CLI showing that you have successfully run a model:
![image](https://user-images.githubusercontent.com/54634163/160420144-5ff9748f-4c19-46a1-b1a2-ca9ceca65252.png)

**Comment by ps-19** [comment]
> @1218muskan i have windows 10 and i installed ubuntu but i faced a problem ![c ubuntu](https://user-images.githubusercontent.com/92082003/160408218-24f25c43-ed96-4e50-bf22-5be66320d398.PNG) Can you tell me hoe you run the program successfully?

You need to enable virtualisation in you PC to use window subsystem with ubuntu.

**Comment by Rufaida98** [comment]
i installed ubuntu and did all the steps right but still the same problem arises

**Comment by prtk2001** [comment]
Hi, I'm Prateek Singh, i'm 3rd year undergraduate from India
    System (OS, Windows., Linux) and version: Ubuntu 20.4 on VMware
    Your background in Python (required skill for this project):Being a machine learning engineer, i'm pretty familiar with python. I have greate experience in tensorflow and a rookie in AWS
    A snapshot of your CLI showing that you have successfully run a model:

![ersilia](https://user-images.githubusercontent.com/56451654/160449203-c28e95d5-8fa4-4895-acf3-608cb91cb2fe.png)

**Comment by ps-19** [comment]
@Rufaida98 can you please attach a screenshot of installation error.

**Comment by ps-19** [comment]
or copy paste the issue.

**Comment by Rufaida98** [comment]
@ps-19 it is as same as above. It came always after I try code "pip install - e." or if i succeeded with that it does not let do the last step which is "ercilia - - help".

**Comment by dauinh** [comment]
Hi @GemmaTuron 

- **Your name**: My name is Linh /ling/ from Vietnam

- **System (OS, Windows., Linux) and version**: MacOS Mojave, version 10.14.6

- **Your background in Python (required skill for this project)**: I'm fairly experienced with Python and familiar with libraries like numpy, matplotlib. I'm experimenting with PyTorch and building deep networks. 

- **A snapshot of your CLI showing that you have successfully run a model**:
![Screen Shot 2022-03-28 at 8 31 28 PM](https://user-images.githubusercontent.com/62679553/160509410-aa6dc004-2b12-4c42-a50c-840d29bebf25.png)

**Comment by Amna-28** [comment]
- Hi, I am **Amna Ali**, a Data science/ Machine Learning enthusiast with masters in Computer Science.
- **System :**  Ubuntu 20.04.3 LTS
- **Python Background :** I have hands-on experience in python as my MS research work was ML/AI related problem. I have worked extensively with the following python libraries and tools :
> - numpy
> - pandas
> - tensorflow
> - matplotlib
> - seaborn
> - scikit-learn
> - pytorch

- A snapshot of your CLI showing that you have successfully run a model

![image](https://user-images.githubusercontent.com/98670206/160511214-ba6c2ede-1963-4b95-8b9e-872ac1cb7524.png)

**Comment by Rufaida98** [comment]
Finally everything works for me.Thank you for everyone who reached to me

**Comment by Rufaida98** [comment]
My name is Rufaida Ali
Windows 10 with Ubunto 18.04
I am so passionate about ML and AI and aspiring to be a data scientist.I am currently learning python and reached
intermediate level.build some game projects
![Capture ersilia](https://user-images.githubusercontent.com/92082003/160561543-f4171c67-ba08-4b31-8530-89a8f6d34629.PNG)

**Comment by Amna-28** [comment]
Hi @miquelduranfrigola @gemmaturon !
I have completed the installation and have run the model.
I was going through the issues but it seems like every issue related to code has been assigned.
Can you please suggest any issue that I could work on? 
I really want to contribute to this project. Thank you.

**Comment by miquelduranfrigola** [comment]
> Hi @miquelduranfrigola @GemmaTuron ! I have completed the installation and have run the model. I was going through the issues but it seems like every issue related to code has been assigned. Can you please suggest any issue that I could work on? I really want to contribute to this project. Thank you.

Thanks @Amna-28! I am now reviewing issues, will update you soon.

**Comment by sonali-sk** [comment]
Hi @GemmaTuron,
> * I am Sonali S K
> * System (OS, Windows., Linux) and version: Windows 10, WSL with Ubuntu 20.04 LTS
> * Your background in Python (required skill for this project): In 2020, I had taken 'Python for Data Science and AI' course by IBM. Since then, Python has been my preferred language to code, and I've used the same in several coding competitions. 
I have explored most of the data structures and am familiar with libraries such as numpy, pandas, matplotlib etc.
> * A snapshot of your CLI showing that you have successfully run a model
![RunSuccess](https://user-images.githubusercontent.com/65673196/160838854-4a2e5dc0-96fb-4519-9983-9921809aeb51.JPG)

**Comment by mahamtariq58** [comment]
Hi @GemmaTuron @miquelduranfrigola  !

- I am Maham Tariq
- My system specfications are : Ubuntu 20.04 LTS 
- I have the basic understanding of Python and have also explored some libraries such as pandas, matplotlib and numpy. I am still learning to increase my expertise in Python. 
-  Snapshot of ersilia CLI
![image](https://user-images.githubusercontent.com/11984160/161272108-676030ce-2cc2-4d6f-95f8-528330880803.png)

**Comment by suryaremanan** [comment]
Hi @GemmaTuron and @miquelduranfrigola, I would like to contribute to the coding project.

<ul>
<li> My Name: Kumari Surya Remanan </li>


<li> System Specs: Ubuntu 20.04 LTS</li>
<li> I am well versed with almost all the machine learning and deep learning algorithms</li>
<li> Snapshot of Ersilia CLI</l>

![Screenshot from 2022-04-02 02-44-46](https://user-images.githubusercontent.com/17148269/161343454-196482c3-7586-4717-a126-c33da96a531e.png)
</ul>

**Comment by suryaremanan** [comment]
@GemmaTuron and @miquelduranfrigola kindly assign me an issue to work upon. Thanks in advance.

**Comment by GemmaTuron** [comment]
Hello all, 

In an effort to improve tracking each applicant work, we have reformatted the contribution schema. Please head over to github and follow the instructions! I will close this issue here to avoid duplicating instructions.
Thanks!

**Comment by Nawarrr** [comment]
- My Name is Abdelrahman Nawar
- **System :**  Pop!_OS 21.04 (Based on Ubuntu 21.04)
- I've been using Python for the past 3 years, I've freelanced doing automations and scripting with python for a couple of months, I also used it to build backends for some projects, It's been the main language I use for the past 3 years and it will probably stay like that as I really love using it
![WhatsApp Image 2022-04-03 at 4 09 41 PM](https://user-images.githubusercontent.com/46817360/161432205-722c3f4a-9255-46ce-8c1e-7c06517c3c72.jpeg)
